packages/express/src/errors.ts

```diff
@@ -0,0 +1,29 @@
+// https://github.com/clerk/javascript/blob/main/packages/remix/src/errors.ts#L1-L0
```

Comment:
That comment isn't helpful, is it?

Reply:
it's a indication that the `createErrorMessage` was copied from the remix package in order to replace it with a shared util but i will remove it since we will probably tackle the error handling in a common way for all packages so there is no need to check this use that comment.